includes/events_data_not_fresh.js

```diff
@@ -1,6 +1,8 @@
 module.exports = (params) => {
   if (!Number.isInteger(params.eventsDataFreshnessDays) || params.eventsDataFreshnessDays < 1) {
     throw new Error(`eventsDataFreshnessDays parameter is not a positive integer.`);
+  } else if (params.eventsDataFreshnessDisableDuringRange && params.disableAssertionsNow) {
+
```

Comment:
Should we have a note explaining why this is blank like we do in the 'entity_data_not_fresh' script? may not be necessary if the users of this code will all understand this